index.js

```diff
@@ -317,26 +317,25 @@ RachioPlatform.prototype.updateZoneAccessory = function(accessory, zone) {
 
     service
         .getCharacteristic(Characteristic.InUse)
-        .on('get', function(callback) {
+        .on('get', async function(callback) {
             logger.debug("get InUse value for " + accessory.UUID)
-            var isWatering = client.getZone(accessory.UUID)
+            var isWatering = await client.getZone(accessory.UUID)
                 .then(zone => zone.isWatering())
```

Comment:
Ideally we would cache the active zone per device (upon startup an onSet) instead of making an async API call on every get.